src/Sulu/Bundle/ContactBundle/Tests/Unit/IndexComparatorTest.php

```diff
@@ -36,7 +39,7 @@ public function usortProvider()
     /**
      * @dataProvider usortProvider
      */
-    public function testUsortCallback($array, $ids, $startsWith, $contains): void
+    public function testUsortCallback(array $array, array $ids, $startsWith, $contains): void
```

Comment:
This is not a bc break because the function where those variables are used already have a return type of array. So the only thing is we fail the code faster on broken types.